aviary/interface/default_phase_info/simple.py

```diff
@@ -36,6 +36,7 @@
             "fix_duration": False,
             "initial_bounds": ((0.0, 0.0), "min"),
             "duration_bounds": ((64.0, 192.0), "min"),
+            "add_initial_mass_constraint": False,
```

Comment:
Why are we making this false? If I remember correctly, this is the constraint that ensures the GTOW used in the pre-mission is the same as the initial mass? Seems important...

Reply:
Ahh gotcha, that's what I get for remembering :)